### ISSUE TITLE
Cannot Suppress  CommentSpacing, NoMultipleSpaces

### ISSUE THREAD

**Issue by radzio** [arturbosch/detekt#1999]
## Expected Behavior
When use `@Suppress("NoTrailingSpaces", "CommentSpacing", "MaxLineLength", "NoMultipleSpaces")` for class all issues "NoTrailingSpaces", "CommentSpacing", "MaxLineLength", "NoMultipleSpaces" should be suppressed

## Observed Behavior

- `NoMultipleSpaces`
- `CommentSpacing`

Are still being reported

## Steps to Reproduce

Code to test

```
@Suppress("NoTrailingSpaces", "CommentSpacing", "MaxLineLength", "NoMultipleSpaces")
class SomeTestClass{

    @ParameterizedTest
    @CsvSource(value = [
        "2019-09-25T10:15:30, , false",                     // some comment
        "2019-09-26T10:15:30, , true",                      // some comment
        "2019-09-27T10:15:30, 2019-09-26T10:15:30, false",  // some comment
        "2019-10-03T04:15:30, 2019-10-02T03:17:30, true",   // some comment
        "2019-10-03T06:12:30, 2019-10-03T04:15:30, false",  // some comment
        "2019-10-15T06:12:30, 2019-10-03T04:15:30, true",   // some comment
        "2019-10-15T06:15:30, 2019-10-15T06:12:30, false",  // some comment
        "2019-10-16T06:12:30, 2019-10-15T04:15:30, true",   // some comment
        "2019-10-17T06:12:30, 2019-10-15T04:15:30, true",   // some comment
        "2019-10-17T06:12:30, , true",                      // some comment
        "2019-10-17T06:12:30, 2019-10-16T04:15:30, false"   // some comment
    ])
    fun someTestMethod(a: String, b: String?, c: Boolean) {
    }
}
```

## Context
I wanted to add some spaces to improve readability. 

## Your Environment

* Version of detekt used: 1.0.1 and 1.1.0
* Version of Gradle used (if applicable): 5.5.1
* Operating System and version: Windows 10
* Link to your project (if it's a public repository):

**Comment by elijah-pl** [comment]
I spent some time trying to set up test for this issue (can be found [here](https://github.com/elijah-pl/detekt/commit/c2a989dc993a2741185b6bea6c34b9ca64a34bbd)), but it seems like I am missing something.

During the debug session, it seems like `annotationEntries` can not be found in the compiled `KtFile`.

```kt
fun KtAnnotated.isSuppressedBy(id: RuleId, aliases: Set<String>): Boolean {
    val valid = mutableSetOf(id, "ALL", "all", "All")
    valid.addAll(aliases)
    return annotationEntries // <- Is empty
            .find { it.typeReference?.text in suppressionAnnotations }
            ?.valueArguments
            ?.map { it.getArgumentExpression()?.text }
            ?.map { it?.replace(detektSuppressionPrefixRegex, "") }
            ?.map { it?.replace(QUOTES, "") }
            ?.find { it in valid } != null
}
```

Any hints would be greatly appreciated!

**Comment by BraisGabin** [comment]
I'm not sure, but maybe this issue was fixed by #2013 

@radzio, could you check with detekt 1.2.0?

**Comment by arturbosch** [comment]
Unfortunately we can't suppress findings from `detekt-formatting` due to performance reasons or due to many workarounds we need to introduce. See #2120

**Comment by arturbosch** [comment]
With the upcoming 1.3.1 release you will be able to suppress these issues via a file level annotation `@file:Suppress("CommentSpacing, ...")`.
Unfortunately due to another data structure used for in KtLint and our wrappers we cannot effectively allow suppression on statement or declaration level.
